docs/resource/os.md

```diff
@@ -14,7 +14,9 @@ In case of virtualized environments, this is the operating system as it is obser
 | `os.type` | string | The operating system type. | `windows` | Required |
 | `os.description` | string | Human readable (not intended to be parsed) OS version information, like e.g. reported by `ver` or `lsb_release -a` commands. | `Microsoft Windows [Version 10.0.18363.778]`; `Ubuntu 18.04.1 LTS` | Recommended |
 | `os.name` | string | Human readable operating system name. | `iOS`; `Android`; `Ubuntu` | Recommended |
-| `os.version` | string | The version string of the operating system as defined in [Version Attributes](/docs/resource/README.md#version-attributes). | `14.2.1`; `18.04.1` | Recommended |
+| `os.version` | string | The major version string of the operating system.  Additionally, build numbers can be added in os.build attribute. | `14.2.1`; `18.04.1` | Recommended |
```

Comment:
I think we should not change the definition of `os.version` for the purpose of what is needed here. Usually, versions do not contain the build number, so adding the `os.build` attribute should be enough to capture that additional information.

Also `The major version string ...`: the word _major_ is misleading here. The major version in `14.2.1` would be `14`, and that's not what is meant here.

Reply:
Thanks for pointing out the same. :) I've reverted the description change for os.version now.